crud/common/utils.lua

```diff
@@ -140,7 +140,7 @@ function utils.unflatten(tuple, space_format)
     return object
 end
 
-function utils.extract_key(tuple, key_parts)
+function utils.extract_from_tuple(tuple, key_parts)
```

Comment:
It was my idea? Now I doubt about this, to be honest, because there is [extract_key()](https://www.tarantool.io/en/doc/latest/reference/reference_lua/key_def/#lua-function.key_def_object.extract_key) in the key_def module. Anyway, let's look at those three function:

1. A function to extract one key from another (accepts the key and two index definitions). IOW, key transformation function.
2. A function to extract a key from pk (particular case of the first function).
3. A function to extract a key from a tuple (accepts the tuple and index definition).

The first one is named `extract_from_index()` and it is a bit confusing, as for me, because it extracts values *from a key* (by index definition/object). 'Extract by index' or 'transform key' looks more appropriate here. While 'extract from pk' is okay, because pk is a key.

So, if we want to reach some consistency, a possible variant is the following:

1. `transform_key()`
2. `transform_pk()`
3. `extract_key()`

What do you think?

Currently the commit, as for me, does not reach the goal it declares: "make function's names consistent and make code clear".

(We can don't spin on it now and postpone such changes if you want.)

Reply:
We discussed it verbally and decided to revert commit 

```
commit 77505f83416994962d7ac5c4800a34e37f470a1a
Author: Sergey Bronnikov <sergeyb@tarantool.org>
Date:   Tue Sep 14 21:56:57 2021 +0300

    Rename utils.extract_key() -> extract_from_tuple()
```

Reply:
> It was my idea? Now I doubt about this, to be honest, because there is extract_key() in the key_def module.

See https://github.com/tarantool/crud/pull/181#issuecomment-924531079

Reply:
I see now, when it is in the sharding_key module. Okay, let's keep utils.extract_key() for now.